devices/ITCommander.js

```diff
@@ -0,0 +1,29 @@
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+module.exports = [
+    {
+        zigbeeModel: ['ITCMDR_Contact'],
+        model: 'ITCMDR_Contact',
+        vendor: 'sumju.net',
```

Comment:
Please change vendor to `IT Commander` and remove it from the `description`.

Reply:
Thank you ， changed